File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/rust/ADV-02_R.rs
use std::io::BufRead;
use std::io::Write;

fn main() {
    let mut out = std::io::stdout();
    writeln!(out, "ADV-START").unwrap();
    out.flush().unwrap();

    let mut line = String::new();
    std::io::stdin().lock().read_line(&mut line).unwrap();
    let a: i64 = line.trim().parse::<i64>().unwrap();

    let v: i64 = a + 3;

    writeln!(out, "OBS=V:{}", v).unwrap();
    out.flush().unwrap();
    writeln!(out, "ADV-END").unwrap();
    out.flush().unwrap();
}

File: benchmark/template/programs/cpp/adversarial/ADV-02_R.cpp
#include <cstdio>
#include <iostream>
#include <string>

int main() {
    printf("ADV-START\n"); fflush(stdout);
    std::string line;
    std::getline(std::cin, line);
    long long x = std::stoll(line);
    long long s = x + 3;
    printf("OBS=V:%lld\n", s); fflush(stdout);
    printf("ADV-END\n"); fflush(stdout);
    return 0;
}
